docs/pages/router/web/static-rendering.mdx

```diff
@@ -243,6 +243,21 @@ export default function Page() {
 }
 ```
 
+For this to work correctly, make sure to wrap your root _layout.tsx with `<Head.Provider>`:
```

Comment:
```suggestion
Then, wrap the root layout navigator (**app/_layout.tsx) with `<Head.Provider>`:
```

This context should be concise and meaningful. The developer won't the context about correct/incorrect usage when the come across this for the first time.